spec/bucket_access_control_spec.rb

```diff
@@ -384,23 +360,17 @@
 
             context 'gstorage_bucket_access_control[title0]' do
               subject do
-                chef_run.find_resource(:gstorage_bucket_access_control,
-                                       'title0')
+                chef_run.find_resource(:gstorage_bucket_access_control, 'title0')
               end
 
               # TODO(alexstephen): Implement resourceref test.
```

Comment:
Do these need to be here?  There's no way that's going to become a priority for us, right?  Same with the complex nested property object test.

Reply:
If we're keeping these tests, there's some low-hanging fruit like this that would clean them up immensely.

Honestly, I don't think it's worth the time since you're writing the replacements